docs/gallery/torontonian_tutorial.ipynb

```diff
@@ -0,0 +1,350 @@
+   "metadata": {},
+   "source": [
+    "For sizes $n=2,14$ we will generate random Hermitian matrices and measure the (average) amount of time it takes to calculate their Torontonian. The number of samples for each will be geometrically distributed, with 1000 samples for size $n=2$ and 2 samples for $n=14$. The matrices will be random Gaussian distributed."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 207,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "a0 = 1000.\n",
+    "anm1 = 2.\n",
+    "n = 7\n",
+    "r = (anm1/a0)**(1./(n-1))\n",
+    "nreps = [(int)(a0*(r**((i)))) for i in range(n)]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 208,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[1000, 354, 125, 44, 15, 5, 2]"
+      ]
+     },
+     "execution_count": 208,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "nreps"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The following function generates random Gaussian matrices of dimensions $n$"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 209,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from thewalrus.random import random_covariance\n",
+    "from thewalrus.quantum.conversions import Amat, Xmat\n",
+    "def random_gaussian(N):\n",
+    "    cov = random_covariance(N)\n",
+    "    O = Xmat(N) @ Amat(cov)\n",
+    "    return O"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now let's benchmark the scaling of the calculation with the matrix size testing both the recursive and non-recursive Torontonian implementations"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 229,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "2 8.993864059448242e-06 1.1003971099853516e-05\n",
+      "4 3.6716461181640625e-05 0.0004632843416289421\n",
+      "6 0.0002240123748779297 0.005471977233886718\n",
+      "8 0.0009090900421142578 0.03940914435820146\n",
+      "10 0.0035334269205729166 0.3631999651590983\n",
+      "12 0.013400125503540038 1.759399652481079\n",
+      "14 0.05500078201293945 11.35949969291687\n"
+     ]
+    }
+   ],
+   "source": [
+    "import time\n",
+    "times, timesrec = np.empty(n), np.empty(n)\n",
+    "for ind,reps in enumerate(nreps):\n",
+    "    size = 2*(ind+1)\n",
+    "    A = random_gaussian(size)\n",
+    "    start = time.time()\n",
+    "    for i in range(reps):\n",
+    "        res = tor(A)\n",
+    "    end = time.time()\n",
+    "    timesrec[ind] = (end - start)/reps\n",
+    "    start = time.time()\n",
+    "    for i in range(reps):\n",
+    "        res = tor(A, recursive=False)\n",
+    "    end = time.time()\n",
+    "    times[ind] = (end - start)/reps\n",
+    "    print(2*(ind+1), timesrec[ind], times[ind])"
```

Comment:
I recommend adding a few empty lines inside the for-loop, e.g., between each `end` and `timesrec[ind]` line, to make it easier to read. 

Also, to follow PEP 8 guidelines, operations should be surrounded by spaces: e.g., `2 * (ind + 1)`.

Reply:
Done.